lib/storage/file.js

```diff
@@ -1061,7 +1061,19 @@ File.prototype.getSignedPolicy = function(options, callback) {
 
   makeAuthorizedRequest_.getCredentials(function(err, credentials) {
     if (err) {
-      callback(err);
+      var signingError = new Error('Signing failed. See `error` property.');
+      signingError.error = err;
+      callback(signingError);
```

Comment:
Since this is happening in more than one place, do you think it would be worth consolidating the logic into something re-usable? like.. `callback(new SigningError(err));`?